### ISSUE TITLE
SVG export not well formed

### ISSUE THREAD

**Issue by jrhawley** [higlass/higlass-python#62]
Hi developers,

While creating various displays, I've tried exporting them into SVGs and PNGs using the `Gear icon > Export views as ...` options.
Exporting as an SVG doesn't produce any errors from HiGlass, but when I go to open the file in an external SVG viewer like Firefox, it shows an "XML Parsing Error: not well-formed" error and doesn't load properly.

Opening the SVG file to the appropriate spot, Firefox highlights the following as improperly formed:
```
<image xlink:href="data:image/png;base64,iVBOl...AAAAAASUVORK5CYII="undefinedxmlns:xlink="http://www.w3.org/1999/xlink" width="256" height="256" />
--------------------------------------------------------------------^
```

From what I've found of other SVG/html files, I'm wondering if that attribute should be written as `...K5CYII=" alt="undefined" xmlns:xlink="...` or something similar in your exporting function.
This might be an issue with a package you're using and not something any of the HiGlass components controls directly, I'm not sure.
I'm also not sure if this is related to the other issues you've had with exporting images, like in #57.

It's obviously not an urgent priority, but exporting the views for manipulation in picture editing software is something I've been trying to do, and I'm sure would be helpful for others, as well. 
Thanks

**Comment by flekschas** [comment]
Sorry for the belated response.

Quick question, when did you install `higlass-python`? We had issues with incorrectly formatted SVGs some time ago in hglib (the actual higlass plotting library) but I thought we fixed the issue. Maybe all it needs is to update higlass-python to get the latest version of higlass.

**WARNING** But please wait updating until https://github.com/higlass/higlass-python/pull/64 is merged.

The reason I am asking is because I tested the SVG export via `Export views as SVG` and it works fine when running the PR.

**Comment by jrhawley** [comment]
I had installed `higlass-python` back in January (v0.4.0) when it gave me the error. I just updated to v0.4.1 and tried it out and the issue seems to be fixed. I'm not totally sure if I had an old `hglib` installed or something like that, but since it's working now I can close this issue.

Thanks for the response

**Comment by flekschas** [comment]
Glad to hear it's working!

FYI, when you install `jupyter labextension install higlass-jupyter` it will take the latest version of higlass.js. But after the widget is installed higlass.js doesn't get updated until you reinstall `higlass-jupyter`. Since we published higlass-python v0.4 quite a few new versions of higlass.js were published, which probably fixed the issue.
